modules/Products/QuickCreate.php

```diff
@@ -43,7 +44,43 @@
 $smarty->assign('BASE_CURRENCY', $base_currency);
 //End - add multi currency
 
-$tax_details = getAllTaxes('available');
+//Tax handling (get the available taxes only) - starts
+$smarty->assign('MODE', $focus->mode);
+if ($focus->mode == 'edit') {
```

Comment:
I think this will never happen. AFAIK QuickCreate is ALWAYS creating, you cannot edit in the popup quick create screen.